code/api/src/modules/user/mutations.test.js

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest'
+import express, { response } from 'express'
+import schema from '../../setup/schema'
+import graphqlHTTP from 'express-graphql'
+import { isType } from 'graphql'
+
+describe('user mutations', () => {
+  let server;
+  beforeAll(() => {
+    server = express();
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: true,
+      })
+    )
+  })
+  it('is true', () => {
+    expect(true).toBe(true)
+  })
+
+  it('updates a user style', async() => {
```

Comment:
Fun part of testing in this framework is that database objects (`user` for example) created in tests are added to the database. So if you want to test the creation of a user you have to also delete it, otherwise when the test is re-run it will error because `user with that name already exists`

Reply:
I liked this test; it was very readable.